docs/pages/branding/about.tsx

```diff
@@ -530,15 +530,6 @@ function AboutContent() {
               Join the team
             </Button>
           </div>
-          <Box display={{ xs: 'none', sm: 'block' }}>
```

Comment:
@siriwatknp We could probably delete this image from the repo. Didn't do it so I don't break stuff 😬 Left it to you.